src/libraries/System.Security.Cryptography.Cose/tests/System.Security.Cryptography.Cose.Tests.csproj

```diff
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(NetCoreAppCurrent);net48</TargetFrameworks>
+    <Nullable>enable</Nullable>
+    <StringResourcesPath>../src/Resources/Strings.resx</StringResourcesPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="$(CommonTestPath)System\Security\Cryptography\ByteUtils.cs">
+      <Link>CommonTest\System\Security\Cryptography\ByteUtils.cs</Link>
+    </Compile>
+    <Compile Include="CoseHeaderMapTests.cs" />
+    <Compile Include="CoseMessageTests.DecoseSign1.cs" />
+    <Compile Include="CoseSign1MessageTests.Verify.cs" />
+    <Compile Include="CoseSign1MessageTests.Sign.cs" />
+    <Compile Include="CoseTestHelpers.cs" />
+    <Compile Include="..\src\System\Security\Cryptography\Cose\KnownHeaders.cs">
+      <Link>System\Security\Cryptography\Cose\KnownHeaders.cs</Link>
+    </Compile>
+    <Compile Include="..\src\System\Security\Cryptography\Cose\KnownCoseAlgorithms.cs">
+      <Link>System\Security\Cryptography\Cose\KnownCoseAlgorithms.cs</Link>
+    </Compile>
+    <Compile Include="..\src\System\Security\Cryptography\Cose\KnownHashAlgorithms.cs">
+      <Link>System\Security\Cryptography\Cose\KnownHashAlgorithms.cs</Link>
+    </Compile>
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.IdentityModel.Tokens" Version="6.15.0" />
```

Comment:
Where/why are we using that?

Reply:
Here: 
https://github.com/Jozkee/runtime/blob/e89936dd7d45c7843feb19a143082debdc43b437/src/libraries/System.Security.Cryptography.Cose/tests/CoseTestHelpers.cs#L104-L107

The Base64URL encoder is the only thing used from that package since .NET doesn't have one (only [Base64 encoder](https://docs.microsoft.com/en-us/dotnet/api/system.convert.frombase64string?view=net-6.0)).